website/client/js/controllers/rootCtrl.js

```diff
@@ -24,8 +24,10 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
     $rootScope.$on('$stateChangeSuccess',
       function(event, toState, toParams, fromState, fromParams){
         $rootScope.pageTitle = $state.current.title;
-
-        $window.scrollTo(0, 0);
+        
+        if ($state.current.name !== 'options.social.challenges.detail') {
```

Comment:
@crookedneighbor I saw it, already made the changes! Thanks a lot, I'm learning so much from you guys. I feel like I didn't really do anything here though, so maybe I shouldn't get credit.